services/QuillLMS/client/app/bundles/Teacher/components/classrooms/classroom_student_section.tsx

```diff
@@ -538,79 +500,88 @@ export default class ClassroomStudentSection
     if (!classroom.visible) { return null }
+
     let loginPdfHref = `/teachers/classrooms/${classroom.id}/student_logins`
+
     let download: boolean
-    if (allGoogleStudents) {
+    if (allStudentsAreGoogle) {
       loginPdfHref = googleSetupInstructionsPdf
       download = true
-    } else if (allCleverStudents) {
+    } else if (allStudentsAreClever) {
       loginPdfHref = cleverSetupInstructionsPdf
       download = true
+    } else if (allStudentsAreCanvas) {
+      loginPdfHref = canvasSetupInstructionsPdf
+      download = true
     }
     /* eslint-disable react/jsx-no-target-blank */
-    const loginPdfLink = <a className="quill-button secondary outlined small" download={download} href={loginPdfHref} rel="noopener noreferrer" target="_blank">Download student log-in instructions</a>
+    const loginPdfLink = (
+      <a
+        className="quill-button secondary outlined small"
+        download={download}
+        href={loginPdfHref}
+        rel="noopener noreferrer"
+        target="_blank"
+      >
+        Download student log-in instructions
+      </a>
+    )
     /* eslint-enable react/jsx-no-target-blank */
 
     return (
       <div className="students-section-header-buttons">
         <div className="login-pdf-and-view-as-student top-buttons-container">
           {loginPdfLink}
-          <button className="quill-button secondary outlined small" onClick={this.handleClickViewAsStudentButton} type="button">
+          <button className="quill-button secondary outlined small" onClick={handleClickViewAsStudentButton} type="button">
             View as student
           </button>
         </div>
-        {this.renderInviteStudents()}
+        {renderInviteStudents()}
       </div>
     )
   }
 
-  renderInviteStudents() {
-    const { classroom, inviteStudents, importCleverClassroomStudents, importGoogleClassroomStudents } = this.props
+  const renderInviteStudents = () => {
+    const { classroomProvider } = classroom
 
     if (!classroom.visible) { return null }
 
-    if (classroom.clever_id) {
+    if (importProviderClassroomStudents) {
       const lastUpdatedDate = moment(classroom.updated_at).format('MMM D, YYYY')
       return (
-        <div className="invite-clever-classroom-students">
-          <button className="quill-button primary outlined small" onClick={importCleverClassroomStudents} type="button">
-            Import Clever classroom students
-          </button>
-          <span>Last imported {lastUpdatedDate}</span>
-        </div>
-      )
-    }
-
-    if (classroom.google_classroom_id) {
-      const lastUpdatedDate = moment(classroom.updated_at).format('MMM D, YYYY')
-      return (
-        <div className="invite-google-classroom-students">
-          <button className="quill-button primary outlined small" onClick={importGoogleClassroomStudents} type="button">
-            Import Google Classroom students
+        <div className="invite-providerclassroom-students">
```

Comment:
This is tiny but you probably wanted a dash between `provider` and `classroom` in this className.

Reply:
Good eye!  There was a styling issue that I was going to fix, and this is probably the culprit.